.github/actions/build-push-image/action.yaml

```diff
@@ -69,28 +74,53 @@ runs:
       uses: docker/setup-buildx-action@v3
       with:
         platforms: ${{ inputs.platforms }}
+        
+    # Devbox setup steps
+    - name: Install devbox
+      uses: jetify-com/devbox-install-action@v0.11.0
+
+    - name: Download Go build dependencies (Devbox)
+      run: devbox run -- go mod download
+      if: ${{ inputs.devbox-build == 'true' }}
```

Comment:
Nit pick, as this IF controls whether the rest runs or not I would place it higher up. I would go for `name` then `if` always:

```yaml
 - name: Some name ...
    if: ${{ inputs.devbox-build == 'true' }}
    ...
```

Reply:
Updated. Thanks!